# destroyObject: throws if onDestroy fails

    Code
      poolReturn(b)
      later::run_now()
    Output
      <pool> Object could not be destroyed but was removed from the pool.
      <pool> Original error:
      Destruction failed...

